sublime_linter.py

```diff
@@ -316,6 +300,10 @@ def highlight(self, view, linters, hit_time):
             panel.fill_panel(window, update=True)
 
         for view in window_views.values():
+            # stop lint indicator
+            ind = persist.indicators[view.id()]
+            ind.stop()
```

Comment:
I dislike short variable names if the abbreviation isn't common (e.g. "img" is fine). This particular one (and a similar one a few lines down) should also work without introducing a variable right?

Reply:
I think those are fine as long as the abbreviation can be deduced from context.
In this case it is clear `ind` stands for *ind*icator. Nonetheless have changed it to a longer variable name. Yes, we could just chain the `start()`/`stop()` method, but I find it less readable.